commands/release.js

```diff
@@ -0,0 +1,231 @@
+const path = require('path');
+const fs = require('fs-extra');
+const chalk = require('chalk');
+const execa = require('execa');
+const npmName = require('npm-name');
+const semver = require('semver');
+
+const GitUtilities = require('../GitUtilities');
+const ConsoleUtilities = require('../ConsoleUtilities');
+const PromptUtilities = require('../PromptUtilities');
+
+const writeJson = (p, json) => fs.writeJson(p, json, { spaces: 2 });
+
+async function getReadme(cwd) {
+  return (await fs.readdir(cwd)).find(p =>
+    path
+      .basename(p)
+      .toLowerCase()
+      .startsWith('readme'),
+  );
+}
+
+async function getNextVersion(version, currentVersion, preid) {
```

Comment:
release-script not using `npm version` here actually made life hard for me on React-Bootstrap, since it means I couldn't use e.g. the `postversion` hook to rebuild docs.

Reply:
We could run those scripts manually I suppose? That's what Lerna does anyway, maybe just use npm version tho

Reply:
Unless we need something else in the commit, just using npm version is probably pretty good.

I do in fact use `np` for all of my OSS stuff though

Reply:
the one thing np doesn't do i use a lot is the `publishDir` thing. maybe i can use np as a dep tho?

Reply:
ah okay i never used that. if we're using it then 👋 `np`